SECURITY.md

```diff
@@ -2,7 +2,7 @@
 
 ## Supported Versions
 
-Information about different FIDO Device Onboard (FDO) release versions can be found on the [FDO Release page](https://secure-device-onboard.github.io/docs/latest/releases/).
+Information about different FIDO Device Onboard (FDO) release versions can be found on the [FDO Release page](https://fido-device-onboard.github.io/docs/latest/releases/).
 
 ## Reporting a Vulnerability
 
```

Comment:
In Line 9 , the hyperlink is https://wiki.lfedge.org/display/SDO/Reporting+Issues , it has SDO in it can it be modified